scipy/ndimage/filters.py

```diff
@@ -274,6 +305,9 @@ def gaussian_filter(input, sigma, order=0, output=None,
     truncate : float
         Truncate the filter at this many standard deviations.
         Default is 4.0.
+    weights : ndarray
```

Comment:
You could point people to correlate1d and your new calculation function in See Also